opta/amplitude.py

```diff
@@ -32,6 +32,7 @@ class AmplitudeClient:
     VIEW_OUTPUT_EVENT = "VIEW_OUTPUT"
     FORCE_UNLOCK_EVENT = "FORCE_UNLOCK"
     FINISH_GEN_EVENT = "FINISH_GEN"
+    GEN_TERRAFORM = "GEN_TERRAFORM"
```

Comment:
Add a FINISH_GEN_TERRAFORM_EVENT, and use it like FINISH_GEN_EVENT so that we can count how many successes/failures there are